src/registrar/templates/django/admin/includes/user_detail_list.html

```diff
@@ -0,0 +1,30 @@
+{% load i18n static %}
+
+{% with approved_domains_count=user.get_approved_domains_count %}
+    {% with active_requests_count=user.get_active_requests_count %}
+        {% with rejected_requests_count=user.get_rejected_requests_count %}
+            {% with ineligible_requests_count=user.get_ineligible_requests_count %}
+                {% if approved_domains_count|add:active_requests_count|add:rejected_requests_count|add:ineligible_requests_count > 0 %}
```

Comment:
Tough to read code because of the limitations of templates.  To assign multiple variables with 'with' statements, need to embed them, can't be combined in one 'with' statement.  The if statement tests for at least one count greater than 0.  I assigned variables because they are used multiple times and I did not want functions executed more than once